src/sentry/static/sentry/app/components/events/eventEntries.tsx

```diff
@@ -290,7 +313,12 @@ const BorderlessEventEntries = styled(EventEntries)`
   }
 `;
 
-const StyledEventUserFeedback = styled(EventUserFeedback)`
+type StyledEventUserFeedbackProps = React.ComponentProps<typeof EventUserFeedback> & {
```

Comment:
you don't need it... you only need `{includeBorder: boolean}`, but please make sure that this prop will not be in the dom 😉